dev/tests/integration/testsuite/Magento/Downloadable/Model/Product/TypeTest.php

```diff
@@ -119,7 +119,7 @@ public function testSaveTypeSpecificData()
         $expectedLink = [
             'is_shareable' => '2',
             'link_type' => 'file',
-            'link_file' => '/j/e/jellyfish_2_4.jpg',
+            'link_file' => '/j/e/jellyfish_2_4_1.jpg',
```

Comment:
@ravi-chandra3197 why did expected link file name should have `_1` suffix?

Reply:
Hello @sivaschenko 
whenever adding the file with the same name that time expectedLink modify because default file system append `_1` with the file name that's why Travis is fail

Reply:
@ravi-chandra3197 I'd ensure that the file is added to the filesystem only once during the integration test suite run or use a unique filenames